source/ClientRequest/AMDSClientStartTimePlusCountDataRequest.h

```diff
@@ -26,7 +26,7 @@ Q_OBJECT
 	/// Sets the start time for the data request
 	inline void setStartTime(const QDateTime &startTime) { startTime_ = startTime; }
 	/// Sets the number of data points after the start time to collect
-	inline void setCount(quint64 count) { count_ = count; }
+	inline void setCount(quint64 count) { count_ = (count==0 ? 1:count) ; }
```

Comment:
This could also probably be replaced with the earlier mentioned method of `count_ = qMax(1, count);`.